README.md

```diff
@@ -63,6 +63,9 @@ OpENer also now has a real-time capable POSIX startup via the OpENer_RT option,
 If you want to use OpENer_RT, instead of step 2, the  ``sudo setcap cap_net_raw,cap_ipc_lock,cap_sys_nice+ep ./src/ports/POSIX/OpENer
 `` has to be run to grant OpENEr ``CAP_SYS_NICE``, ``CAP_IPC_LOCK``, and the ``CAP_NET_RAW`` capabilities, needed for the RT mode
 
+Shared library support has been added to CMakeLists file and is enabled by setting BUILD_SHARED_LIBS=ON. It has only been tested under Linux/POSIX platform.
```

Comment:
In my case, I have an outer CMakeLists that sets it. Being fairly new to CMake, I'm not sure I  understand the implications of "hiding" and "caching" the variable, and how it might impact my build setup.

Reply:
Oh, ok. You can add CMake variables, which show not up in the CMake configurator. E.g., OpENer_CIP_OBJECTS_DIR is an CMake variable, but not presented to the user for direct manipulation.

OpENer_TRACES is a cached variable, so it will be shown in the CMake configuration for the user to modify.

Your BUILD_SHARED_LIBS should probably be a cached variable, so it can be selected from a CMake configurator, would you agree?

Reply:
Yes.